packages/accounts-base/accounts_client_tests.js

```
@@ -147,6 +147,126 @@ Tinytest.addAsync(
+      });
+      Meteor.logout();
+    });
+  }
+);
+
+Tinytest.addAsync(
+  'accounts async - async userCallback completes its async work',
+  (test, done) => {
+    logoutAndCreateUser(test, done, () => {
+      Meteor.logout(() => {
+        Meteor.loginWithPassword(username, password, async (err) => {
```

ESLint

[error] Replace `(err)` with `err`

(prettier/prettier)